### PR TITLE
Speedup cluster creation by using pre-generated certs.

### PR DESCRIPTION
**Pull request by petermattis**
This reduces the total run time of the acceptance tests from 42sec to
27s on my machine.

Added TestCertGen which generates new certs and verifies the nodes can
talk to each other.

Rename the "-n" flag to "-num" to avoid a conflict with the "-n" flag in
the go tool.

Regenerated the test_certs to contain alternate names for
"roach{0-99}.local".

### CODE REVIEW

**Comment by MBerhault** [comment]
how are they taking so long? I'm at 1.5-2 seconds of cert stuff for each test case for a total runtime of 28s

**Comment by MBerhault** [comment]
LGTM

**Comment by petermattis** [comment]
@mberbault strange, cert gen takes ~5 sec per test on my box.

```
=== RUN TestCertGen
I0522 11:03:57.545203   69036 localcluster.go:195] started docker-spy: 172.17.5.115
I0522 11:03:57.545359   69036 localcluster.go:201] initializing cluster
I0522 11:03:58.196662   69036 localcluster.go:297] creating ca
I0522 11:04:01.103173   69036 localcluster.go:305] creating node certs
I0522 11:04:03.597022   69036 localcluster.go:335] started roach0.local: 192.168.59.103:33921
I0522 11:04:04.515067   69036 localcluster.go:335] started roach1.local: 192.168.59.103:33922
```

**Comment by MBerhault** [comment]
Nasty. I'm guessing boot2docker adds a lot of overhead.
Yet another reason to do benchmarks outside VMs.

**Comment by bdarnell** [comment]
It could be a lack of entropy in the VM instead of "overhead".

File: acceptance/localcluster/localcluster.go [hunk 1]
```diff
@@ -40,6 +41,7 @@ const (
 var cockroachImage = flag.String("i", builderImage, "the docker image to run")
 var cockroachBinary = flag.String("b", defaultBinary(), "the binary to run (if image == "+builderImage+")")
 var cockroachEntry = flag.String("e", "", "the entry point for the image")
+var testCertsDir = filepath.Clean(os.ExpandEnv("${PWD}/../resource/test_certs"))
```
Comment (bdarnell) [inline acceptance/localcluster/localcluster.go]: `${PWD}` is funky here: this works for tests in the `acceptance` package but wouldn't work for localcluster's own tests (if it had any). Not sure it's worth fixing at the moment but we could have run.sh pass in the cert path.

**Pull request merged**